README.md

```diff
@@ -1,47 +1,52 @@
 <img src="https://raw.githubusercontent.com/elbb/registry-cleanup/master/.assets/logo.png" height="200">
 
 # registry-cleanup
+
 This interactive tool allows a user to delete images and/or single tags from a docker registry. It is written in golang.
 
 ## Requirements
 
-* golang 1.13
+-   golang 1.13
 
 ### Building
+
 Clone the repo wherever you like and build it using 
 ` $ make registry-cleanup`
 
 You find the binary located in `$GOBIN`
 
 ## Usage
 
-```
-./registry-cleanup -h
-Usage of registry-cleanup
+    $GOBIN/registry-cleanup -h
+    Usage of registry-cleanup
 
-Usage:
-  -address string
-    	docker registry URI (default "http://localhost:5000")
-  -user string
-    	User for auth to docker registry, empty for anymous
-```
+    Usage:
+      -address string
+        	docker registry URI (default "http://localhost:5000")
```

Comment:
there is no default? I can not use it without explicitly writing 
`-address ...`